cubedash/templates/500.html

```diff
@@ -15,13 +15,4 @@ <h4>Internal error</h4>
     This has been logged.
 </div>
 </body>
-{% if sentry_event_id %}
```

Comment:
Should add new JS snippet here : https://docs.sentry.io/platforms/javascript/install/cdn/ . Creating separate front end fix issue for it.